c2corg_common/attributes.py

```diff
@@ -667,3 +668,85 @@
     'aran',
     'catalunya'
 ]
+
+event_types = [
+    'avalanche',
+    'stone_fall',
+    'falling_ice',
+    'person_fall',
+    'crevasse_fall',
+    'roped_fall',
+    'physical_failure',
+    'lightning',
+    'other'
+]
+
+author_statuses = [
+    'primary_impacted',
+    'secondary_impacted',
+    'internal_witness',
+    'external_witness'
+]
+
+activity_rates = [
+    'activity_rate_150',
```

Comment:
Using more explicit values would make sense (by the way how translators can figure out how to translate this? :P ). Ditto for ``nb_outings``. I know those are the values from v5.

@stef74 @fbunoz can you provided less confusing values?
If you think we can live with them, OK for me.